assets/js/components/ReportError.js

```diff
@@ -69,7 +69,7 @@ export default function ReportError( { moduleSlug, error } ) {
 			err.selectorData.name === 'getReport'
 	);
 
-	const showRetry = !! retryableErrors.length && ! isViewOnly;
+	const showRetry = !! retryableErrors.length;
```

Comment:
I realise now that [this change](https://github.com/google/site-kit-wp/pull/6292/files#diff-b95a49b3fe9c5fd7c5b5f782eef76c5c71fd74299f4bdce4a7156144db48d621L71-R72) was most likely unnecessary as the ACs do not mention anything about removing the "Retry" actions for view-only users, and I don't think we need to remove them anyway for view-only users. This is why, I've reverted this change with this PR.